osiguranje na klik/homepage.html

```diff
@@ -55,108 +55,154 @@
+            <a href="" class="logo-dl"><img src="img/logo-sava.png" alt="" ></a>
+            <a href="" class="logo-dl"><img src="img/logo-dunav.png" alt="" ></a>
+            <a href="" class="logo-dl"><img src="img/logo-triglav.png" alt="" ></a>            
+            <a href="" class="logo-dl"><img src="img/logo-wiener-stadtische.png" alt="" ></a>
+            <a href="" class="logo-dl"><img src="img/logo-ddor.png" alt="" ></a>
+            <a href="" class="logo-dl"><img src="img/logo-generali.png" alt="" ></a>
+        </div>
+    </section>
+    <section class="quote-bg">
+        <div class="quote max-width">
+            <h1 class="quote-title">Šta kažu naši korisnici</h1>
+            <p class="quote-review">Pronašao sam ovaj sajt i shvatio da sve mogu da završim online. Uštedeo sam i vreme da završim važne obaveze pred put. Veoma sam zadovoljan brzom uslugom i dobrom polisom.</p>
+            <p class="quote-user">Stefan Marković</p>
+            <p class="quote-user-profession">Programer</p>
+            <div class="quote-actions">
+                <a href="" class="quote-left-arrow arrow-container"><img src="img/Arrow Left.png" class="prev" alt=""></a>
+                <a href="" class="quote-indicator"><img src="img/active.png" alt=""></a>
+                <a href="" class="quote-indicator"><img src="img/inactive.png" alt=""></a>
+                <a href="" class="quote-indicator"><img src="img/inactive.png" alt=""></a>
+                <a href="" class="quote-indicator"><img src="img/inactive.png" alt=""></a>
+                <a href="" class="quote-right-arrow arrow-container"><img src="img/Arrow Right.png" class="next" alt=""></a>
+            </div>
+        </div>
+    </section>
+    <section class="nl-container max-width">
         <div class="newsletter">
             <div class="nl-text-container">
-                <p class="nl-title">Prijavite se na naš newsletter</p>
+                <h1 class="nl-title">Prijavite se na naš newsletter</h1>
                 <p class="nl-description">Budite u toku sa novostima i savetima iz sveta osiguranja.</p>
             </div>                 
             <div class="nl-input">
                 <input class="nl-input-field" type="email" placeholder="Vaša mejl adresa">
                 <button class="nl-input-field-button"><img src="img/Right-wihite-arrow.png" alt=""></button>
             </div>
         </div>
-    </div>
-    <div class="footer-container">
+    </section>
+    <section class="footer-container">
         <div class="footer max-width">
             <div class="ft-links">
                 <div class="ft-box">
-                    <p class="ft-title">Vrste osiguranja</p>
+                    <h2 class="ft-title">Vrste osiguranja</h2>
                     <a href="">Putno osiguranje</a>
                     <a href="">Osiguranje imovine</a>
                     <a href="">Osiguranje vozila</a>
```

Comment:
Praksa za ove linkove je da budu unutar liste, posto i jeste list.

```suggestion
                <ul>
                  <li>
                    <a href="">Putno osiguranje</a>
                  </li>
                  <li>
                    <a href="">Osiguranje imovine</a>
                  </li>
                  <li>
                    <a href="">Osiguranje vozila</a>
                  </li>
                </ul>
```